test/tactics.lean

```diff
@@ -203,6 +203,12 @@ begin
   tfae_finish -- the success or failure of this tactic is nondeterministic!
```

Comment:
Is this comment still relevant? Doesn't inspire confidence...
But apparently it has passed several hundred tests without failing :thinking:

Reply:
No, that example came from a bug report that got fixed.